Test/TestMzML.cs

```diff
@@ -586,7 +586,7 @@ public void Setup()
         [Test]
         public void LoadMzmlTest()
         {
-            Assert.Throws<AggregateException>(() =>
```

Comment:
the AppVeyor build is failing because of this unit test - try making it just Exception instead of MzLibException or AggregateException